content/en/tracing/services/service_page.md

```diff
@@ -149,6 +149,13 @@ This tab has overview graphs that show which resources have the most issues and
 
 {{< img src="tracing/visualization/service/error_tracking_side_panel.jpg" alt="Error Tracking tab"  style="width:90%;">}}
 
+### Security
+View security signals on your service, which are automatically created when Datadog detects application attacks impacting your services, identifying meaningful threats to review instead of assessing each individual attack attempt. Read more about [Application Security][18].
```

Comment:
```suggestion
View security signals on your service, which are automatically created when Datadog detects application attacks impacting your services. The signals identify meaningful threats for you review instead of assessing each individual attack attempt. Read more about [Application Security][18].
```